File: my/src/main/java/com/littlecorgi/my/logic/model/JoinClassResponse.java
package com.littlecorgi.my.logic.model;

import java.io.Serializable;
import lombok.Data;
import lombok.NoArgsConstructor;

/**
 * 加入班级的响应
 *
 * @author littlecorgi 2021/5/4
 */
@NoArgsConstructor
@Data
public class JoinClassResponse implements Serializable {

    private static final long serialVersionUID = 1234567890605L;

    private Integer status;
    private String msg;
    private String errorMsg;
    private long data;
}

File: my/src/main/java/com/littlecorgi/my/logic/model/JoinClassResponse.kt
package com.littlecorgi.my.logic.model

import java.io.Serializable

/**
 * 加入班级的响应
 *
 * @author littlecorgi 2021/5/24
 */
data class JoinClassResponse(
    var status: Int? = null, // 状态码
    var msg: String? = null, // 信息
    var errorMsg: String? = null, // 错误信息
    var data: Long? = null // 内容
) : Serializable {
    companion object {
        private const val serialVersionUID = 5990939387657230604L
    }
}